action.yml

```diff
@@ -93,6 +96,27 @@ inputs:
   language-framework:
     description: "The language framework of the container."
     required: false
+  enable-liveness-probe:
+    description: "If false, will disable the liveness probe of the app instance.  Allowed values: false, true."
+    required: false
+  enable-readiness-probe:
+    description: "If false, will disable the readiness probe of the app instance.  Allowed values: false, true."
+    required: false
+  enable-startup-probe:
+    description: "If false, will disable the startup probe of the app instance.  Allowed values: false, true."
+    required: false
+  termination-grace-period-seconds:
+    description: "Optional duration in seconds the app instance needs to terminate gracefully."
+    required: false
+  liveness-probe-config: 
+    description: "A json file path indicates the liveness probe config"
+    required: false
+  readiness-probe-config: 
+    description: "A json file path indicates the readiness probe config"
```

Comment:
Does GitHub Actions supports object arguments? In that case we may write the config as

```
rediness-probe-config:
   httpGet:
      url: ....
```

Reply:
We don't support object arguments right now. CLI only support json file arguments so we just keep the feature parity here.

Reply:
After investigation, GitHub Action only support three get input method. We cannot get an object from yaml
```
export declare function getInput(name: string, options?: InputOptions): string;
export declare function getMultilineInput(name: string, options?: InputOptions): string[];
export declare function getBooleanInput(name: string, options?: InputOptions): boolean;
```
Here is also one question about how to get object but no solution. https://stackoverflow.com/questions/67050295/passing-object-as-a-variable-to-github-action
Here is the input metadata syntax doc: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs